crates/wasm_interp/tests/test_f64.rs

```diff
@@ -120,5 +158,28 @@ fn test_f64div() {
 
     test_f64_binop(op, 1.0, 0.0, f64::INFINITY);
     test_f64_binop(op, -1.0, 0.0, f64::NEG_INFINITY);
-    // test_f64_binop(op, 0.0, 0.0, f64::NAN); // can't check NaN for equality! LOL
+    // to-probably-never-do: check NaN. It needs its own special test setup.
```

Comment:
lol, what is the problem here?

Reply:
That's it, yes, should have been more explicit!